lib/src/command/add.dart

```diff
@@ -39,31 +39,18 @@ class AddCommand extends PubCommand {
   String get description => 'Add dependencies to pubspec.yaml.';
```

Comment:
```suggestion
  String get description => '''
Add dependencies to `pubspec.yaml`.

Invoking `dart pub add foo bar` will add `foo` and `bar` to `pubspec.yaml`
with a default constraint derived from latest compatible version.
''';
```

`Command.summary` will be derived from the first-line, otherwise if we want to customize the summary, we can override it.

But I think a command with a usage this complicated will benefit greatly from examples in `--help`. Especially, since most users will be interested in doing `dart pub add <package>` or `dart pub add <package>:<constraint>`

Reply:
Done